services/frontend/src/components/profileCards/employmentStatusCard/EmploymentStatusCardView.jsx

```diff
@@ -3,6 +3,27 @@ import { Col, List, Row } from "antd";
 
 import { ProfileInfoPropType } from "../../../utils/customPropTypes";
 
+/**
+ * Retrieves the ribbon button for adding/removing a user from one's circle,
+ * or the edit button which is a link to the edit profile page
```

Comment:
The description for this component appears to be for the `ActionRibbonBtn` component and looks like it should be changed.